querydsl-core/src/main/java/com/querydsl/core/types/QBean.java

```diff
@@ -86,6 +86,10 @@ private static boolean isCompoundExpression(Expression<?> expr) {
         return cl.isPrimitive() ? Primitives.wrap(cl) : cl;
     }
 
+    private static boolean isAssignableFrom(Class<?> cl1, Class<?> cl2) {
+        return normalize(cl1).isAssignableFrom(normalize(cl2));
```

Comment:
Is it maybe an idea to move `ConstructorUtils.compatible` together with `normalize` to somewhere else, or make it public?
QBean can then also use it, since there is right now duplication.